www/src/components/landingPage/community/community.astro

```diff
@@ -7,80 +9,68 @@ export interface Props {}
   <div
     class="flex flex-col items-center justify-center max-w-7xl text-t3-purple-100 mx-auto"
   >
-    <h1 class="text-3xl py-3">Community</h1>
+    <h1 class="text-3xl py-3 font-bold">Community</h1>
     <p class="text-center pb-8 px-4">
       Join our community to get help, share your projects, and even contribute
       to the project!
     </p>
     <div
       class="flex flex-col sm:flex-row items-start justify-center space-y-4 sm:space-y-0 sm:space-x-6 mx-2 sm:mx-4 lg:mx-8"
     >
-      <a
+      <CommunityCard
+        title="Discord"
+        text="Join our Discord server to chat with other developers and get help with your projects."
         href="https://t3.gg/discord"
-        target="_blank"
-        rel="noopener noreferrer"
-        class="flex flex-col items-start justify-center border border-t3-purple-100 rounded-lg p-3 space-y-3 h-48 sm:h-56 hover:bg-white/10 cursor-pointer w-full"
       >
         <svg
           role="img"
           viewBox="0 0 24 24"
           xmlns="http://www.w3.org/2000/svg"
           class="fill-[#5865F2]"
+          slot="icon"
```

Comment:
Astro 1.1.2 is now out which fixes the typing of SVGs. Updating to that will silence the issues and pass CI 👍